src/token.html

```
@@ -98,6 +98,29 @@ <h3 class="text-sm lg:text-base uppercase font-semibold font-headline text-camin
+            {% image "./src/static/images/summer-takeoff/camino-logo-summertakeoff.png", "", "", "mb-5 w-auto h-8 object-contain" %}
+            <div class="text-headline">
+                Hold a token that actually takes you places.
+            </div>
+            <p class="my-4 md:my-6 text-gray-300">
+                This summer, Camino Network is rewarding its community with exclusive travel perks — not for staking, or locking.
+            </p>
+            <p class="mb-4 md:mb-6 text-caminoSky font-bold">
+                Just for being part of it — Hold CAM and you’re in the draw for real travel rewards. 
+            </p>
+            <div class="mt-auto">
+                <a class="button" href="https://app.deform.cc/form/f3d3e2ac-106a-43d5-84ee-1891a0ee1d1e/" target="_blank" rel="noopener noreferrer">Learn more -></a>
```

HTMLHint (1.5.0)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)